packages/app-desktop/gui/ConfigScreen/ConfigScreen.tsx

```diff
@@ -60,12 +61,28 @@ class ConfigScreenComponent extends React.Component<any, any> {
 		this.setState({ settings: this.props.settings });
 	}
 
+	updatePluginsStates(value: any) {
+		if (Object.keys(value).length === 0) return;
+		const key = 'plugins.states';
+		const md = Setting.settingMetadata(key);
+		let newValue = Setting.value('plugins.states');
+		newValue = {
+			...newValue, ...value,
+		};
+		shared.updateSettingValue(this, key, newValue);
+
+		if (md.autoSave) {
+			shared.scheduleSaveSettings(this);
+		}
+	}
+
```

Comment:
This function can also move into a default Plugins module. If you can't move it please leave a detailed comment about why it must be defined.

Reply:
Done :+1: